### PR TITLE
Recent Activity Modal Add

### PR DESCRIPTION
**Pull request by brijsiyag**
https://user-images.githubusercontent.com/71015892/168659459-d2bb6b79-142f-4504-9392-6a14960841a0.mp4

### CODE REVIEW

**Comment by garg3133** [comment]
Instead of putting the link on the title, can you create a button at the bottom with some text like: "Read more" which will redirect you to the linked page? If the link represents an image and you're showing that image on the modal itself, there's no need to the button.

Also, I think it would be better to put the title of the notice on the header of the model instead of putting 'Activity' there. Also, no need to keep the header orange if some shade of grey or just white with a border at the bottom works here, orange looks a bit odd.

And also please put a 'cross' button at the top-right corner of the modal. Since a percentage of users using this application are going to be older, we should provide them with enough visual aesthetics to use the application with ease.

File: .gitignore [hunk 1]
```diff
@@ -9,6 +9,10 @@ db.sqlite3
 media
 staticfiles
 .vscode
+notice_files
+profile_photos
```
Comment (garg3133) [inline .gitignore:13]: Why put these files in `.gitignore`? If these are just some files you are creating on your local to do some tests, you don't need to put these in `.gitignore`, just don't stage those files while committing.

**Review by garg3133 — COMMENTED** [review]
Other than the comments above and one comment below, LGTM!

**Comment by brijsiyag** [comment]
> Instead of putting the link on the title, can you create a button at the bottom with some text like: "Read more" which will redirect you to the linked page? If the link represents an image and you're showing that image on the modal itself, there's no need to the button.
> 
> Also, I think it would be better to put the title of the notice on the header of the model instead of putting 'Activity' there. Also, no need to keep the header orange if some shade of grey or just white with a border at the bottom works here, orange looks a bit odd.
> 
> And also please put a 'cross' button at the top-right corner of the modal. Since a percentage of users using this application are going to be older, we should provide them with enough visual aesthetics to use the application with ease.

Sorry @garg3133, I didn't notice this comment earlier.
Could you please reopen this PR?

**Pull request merged**